test/module/irohad/consensus/yac/yac_gate_test.cpp

```diff
@@ -208,3 +208,50 @@ TEST_F(YacGateTest, LoadBlockWhenDifferentCommit) {
 
   ASSERT_TRUE(gate_wrapper.validate());
 }
+TEST_F(YacGateTest, LoadBlockWhenDifferentCommitFailFirst) {
```

Comment:
The test looks really similar to the previous, maybe it's better refactor those (e.g extract same routines) and simplify the tests code?